lambda_function.py

```diff
@@ -0,0 +1,8 @@
+import json
+
+def my_handler(event, context):
+    print("Hello world")

```

Comment:
The lambda just printing hello world? We need to print the metadata of the s3 file.